getting-started-github-graphql-api.md

```diff
@@ -40,59 +42,59 @@ When interacting with GitHub's GraphQL API, it would be great to fill your own a
 
 {{% sub_chapter_header "GitHub Repositories" "github-repository" %}}
 
-Secondly, you can create repositories on GitHub. You may wonder what's a repository? *"A repository is the most basic element of GitHub. They're easiest to imagine as a project's folder. A repository contains all of the project files (including documentation), and stores each file's revision history. Repositories can have multiple collaborators and can be either public or private."* The quote is taken from {{% a_blank "GitHub's glossary" "https://help.github.com/articles/github-glossary/" %}} which helps you to get to know GitHub's terms (e.g. repository, issue, clone, fork, push) which you may need for the following sections. Basically a repository is the place for your application's source code to share it with others. If you don't have any repositories on GitHub yet, I encourage you to put a few of your project's (source code) into repositories by pushing them to GitHub. Later you can access these repositories with GitHub's GraphQL API.
+You can also create repositories on GitHub. In the words of their official glossary: *"A repository is the most basic element of GitHub. They're easiest to imagine as a project's folder. A repository contains all of the project files (including documentation), and stores each file's revision history. Repositories can have multiple collaborators and can be either public or private."* {{% a_blank "GitHub's glossary" "https://help.github.com/articles/github-glossary/" %}} will explain the key terms--repository, issue, clone, fork, push--which are necessary to follow along with the upcoming sections. Basically a repository is the place for application source code that can be shared with others.  I encourage you to put a few of your projects into GitHub repositories, so you can access them all later with what you've learned about their GraphQL API
```

Comment:
"which are necessary to follow along with the upcoming sections" to "which are necessary to follow along with the upcoming chapters to learn about GraphQL"

Reply:
"with what you've learned about their GraphQL API" to "with what you will learn about their GraphQL API."

Reply:
* I made these changes, but I would advise against continuously using the same phrases (i.e. "learn about GraphQL") to improve SEO.  There comes a point when search engines see enough mentions of a subject and stop counting it toward bumping you up in search results.  In fact, it can trigger Google's anti keyword-stuffing mechanism if it looks like you're trying to trick the system.  The book will AUTOMATICALLY have fantastic SEO for anything related to GraphQL, React, etc. because they will be mentioned 1,000 times naturally.  Adding it unnecessarily just lowers the quality of the reading.
* The second correction is incorrect grammar.  You're probably worried about confusing the reader into thinking they can go there now with with they have learned, but the beginning the sentence reads "so you can access them all LATER with what you've learned"  When you establish that you are talking about the future (which you did by saying "later"), you use the phrase "you have" not "you will".

Reply:
Good points. Thanks for clarification!